pkg/kubectl/cmd/wait/wait.go

```diff
@@ -184,17 +184,19 @@ func conditionFuncFor(condition string) (ConditionFunc, error) {
 	return nil, fmt.Errorf("unrecognized condition: %q", condition)
 }
 
+// ResourceLocation holds the location of a resource
 type ResourceLocation struct {
 	GroupResource schema.GroupResource
 	Namespace     string
 	Name          string
 }
 
+// UIDMap maps ResourceLocation with UID
 type UIDMap map[ResourceLocation]types.UID
 
-// WaitOptions is a set of options that allows you to wait.  This is the object reflects the runtime needs of a wait
+// Options is a set of options that allows you to wait.  This is the object reflects the runtime needs of a wait
 // command, making the logic itself easy to unit test with our existing mocks.
-type WaitOptions struct {
+type Options struct {
```

Comment:
How about `CLIWaitOptions`?

Reply:
Not sure. need more opinions here... I used naming convention as followed [here](https://github.com/kubernetes/kubernetes/blob/b1e15b4d98f55d6c1cbb5da58244790c0a80a8a4/pkg/kubectl/apply/strategy/merge.go#L22) and [here](https://github.com/kubernetes/kubernetes/blob/b1e15b4d98f55d6c1cbb5da58244790c0a80a8a4/pkg/kubelet/server/remotecommand/httpstream.go#L42)